packages/pds/Dockerfile

```diff
@@ -46,7 +46,7 @@ ENV NODE_ENV=production
 
 # https://github.com/nodejs/docker-node/blob/master/docs/BestPractices.md#non-root-user
 USER node
-CMD ["node", "--enable-source-maps", "index.js"]
```

Comment:
this is temporary right? just so we can get better diagnostics?

Reply:
It doesn't have to stick around, but actually I don't know of any downsides of keeping it!